python/data_structures/linked_list.py

```diff
@@ -1,16 +1,49 @@
+
 class LinkedList:
     """
     Put docstring here
     """
 
     def __init__(self):
-        # initialization here
-        pass
+        self.head = None
+
+    def insert(self, value):
+        self.head = Node(value, self.head)
+
+    def __str__(self):
+        null = 'NULL'
+        if self.head is None:
+            return null
+        current = self.head
+        output = ''
+        while current:
+            output += '{ ' + current.value + ' }' + ' -> '
+            current = current.next
+        print(output + null)
```

Comment:
Don't forget to remove unnecessary print statements before merging from a working branch.